qa/workunits/rbd/rbd_mirror_group_simple.sh

```diff
@@ -1358,6 +1358,24 @@ test_empty_group()
   check_daemon_running "${secondary_cluster}"
 }
 
+# test removal of a non existing group
+declare -a test_remove_non_existing_group_1=("${CLUSTER2}" ${pool0} "group_blabla")
+
+test_remove_non_existing_group_scenarios=1
+
+test_remove_non_existing_group()
+{
+  local primary_cluster=$1
+  local pool=$2
+  local non_existing_group=$3
+
+  # command fails with the following message now
+  # rbd: remove error: (2) No such file or directory
+  # 2025-09-25T20:51:08.402+0530 7fe0ea9a9d00 -1 librbd::api::Mirror: group_disable: error getting the group id: (2) No such file or directory
+  # 2025-09-25T20:51:08.402+0530 7fe0ea9a9d00 -1 librbd::api::Group: remove: failed to disable mirroring: (2) No such file or directory
```

Comment:
What build did you use for testing?  I'm not following how these error messages could be generated and the overall error could be ENOENT when `GroupGetInfoRequest` would emit "both group name and group id cannot be empty" error message and fail with EINVAL.

Reply:
Updated the test now. I'm not sure what's wrong with my previous old branch, need to investigate it.

Reply:
It looks like the difference between old and new branches are:

```
[root@dhcp53-181 build]# git lo wip-rbd-cgsm-base..fix-segfault 
* 4eaf21d882b qa/workunits: add scenario to "test_force_promote_delete_group"
* 55dfdb8ebb6 rbd-mirror: skip validation of primary demote snapshots
* ff3570d1e12 librbd/api: restrict APIs on secondary mirror groups
* 8afc57408eb librbd/api/Group.cc: use correct error code symbolic
* c6f869ffc9c librbd/api/Mirror: enhancements to group_enable API
```
The old one misses the above commits. And probably is why I got there. Never mind.